<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneryMapSettings</class>
 <widget class="QWidget" name="SceneryMapSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>282</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>152</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>200</width>
        <height>282</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>280</height>
       </size>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>170</width>
         <height>281</height>
        </rect>
       </property>
       <property name="title">
        <string>map</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <item row="5" column="1">
         <widget class="QDoubleSpinBox" name="xBox">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <double>-10000000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="xLabel">
          <property name="text">
           <string>x [m]</string>
          </property>
          <property name="buddy">
           <cstring>xBox</cstring>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="yLabel">
          <property name="text">
           <string>y [m]</string>
          </property>
          <property name="buddy">
           <cstring>yBox</cstring>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QDoubleSpinBox" name="widthBox">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <double>-10000000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QLabel" name="opacityLabel">
          <property name="text">
           <string>opacity
[%]</string>
          </property>
          <property name="buddy">
           <cstring>opacityBox</cstring>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QDoubleSpinBox" name="heightBox">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <double>-10000000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QDoubleSpinBox" name="yBox">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <double>-10000000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="14" column="1">
         <widget class="QSpinBox" name="opacityBox">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>filename</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="dataLabel">
          <property name="text">
           <string>data</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="changeDataButton">
          <property name="text">
           <string>Change File</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="heightLabel">
          <property name="text">
           <string>height [m]</string>
          </property>
          <property name="buddy">
           <cstring>heightBox</cstring>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="widthLabel">
          <property name="text">
           <string>width [m]</string>
          </property>
          <property name="buddy">
           <cstring>widthBox</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="changeFileButton">
          <property name="text">
           <string>Change File</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="filenameText">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="dataText">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
